templates/node/node--search-result.html.twig

```diff
@@ -0,0 +1,84 @@
+{#
```

Comment:
All of these comments can be deleted. If someone wants to know what's available, they can look in the Drupal core template.

Reply:
I removed the comments.